picard/formats/id3.py

```diff
@@ -494,6 +494,19 @@ def _get_tags(self, filename):
         except mutagen.id3.ID3NoHeaderError:
             return compatid3.CompatID3()
 
+    def _load_preserved_config(self, metadata):
+        # Adding additional config to metadata to prevent unecessary saving
+        metadata['~config:rating_steps'] = config.setting['rating_steps']
+        metadata['~config:rating_user_email'] = config.setting['rating_user_email']
```

Comment:
P.S. It seems to me that a list of settings which change media tags without changes to Picard's `metadata` is quite small. Images, ratings, what else? So we create a list of these settings, and preserve the startup values and then compare the startup values to current values when user saves files.

What do people think of this idea?